GitUI/GitUI.csproj

```diff
@@ -1890,6 +1890,6 @@
     <PropertyGroup>
       <ErrorText>This project references NuGet package(s) that are missing on this computer. Use NuGet Package Restore to download them.  For more information, see http://go.microsoft.com/fwlink/?LinkID=322105. The missing file is {0}.</ErrorText>
     </PropertyGroup>
-    <Error Condition="!Exists('..\packages\ConEmu.Core.16.3.10.0\build\ConEmu.Core.Targets')" Text="$([System.String]::Format('$(ErrorText)', '..\packages\ConEmu.Core.16.3.10.0\build\ConEmu.Core.Targets'))" />
+    <Error Condition="!Exists('..\packages\ConEmu.Core.16.3.10.0\Build\ConEmu.Core.Targets')" Text="$([System.String]::Format('$(ErrorText)', '..\packages\ConEmu.Core.16.3.10.0\build\ConEmu.Core.Targets'))" />
```

Comment:
Interesting. This code is autogenerated by Nuget and is never authored manually.
Nuget is case-insensitive in most aspects like package IDs and so on, so it might not be very careful about casing. This could be a nuget own issue.